src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -279,58 +285,84 @@ struct rr_connectivity_data {
   glb_lb_policy *glb_policy;
 };
 
+static bool process_serverlist(const grpc_grpclb_server *server,
```

Comment:
If I'm reading the code right, this function actually processes a single entry in the serverlist, not the serverlist itself.  Perhaps call it something like `extract_server_address()` instead?